agent/CMakeLists.txt

```diff
@@ -25,6 +25,8 @@ add_executable(softwarecontainer-agent
     src/softwarecontaineragent.cpp
     src/softwarecontaineragentadaptor.cpp
     ${GENERATED_ADAPTOR_FILENAME}
+    src/capability/configstore.cpp
+    src/capability/capability.cpp
```

Comment:
I know. Kept partly to start discussion on if I should use it. If no reason pops up for using it, I will remove it.